interview.md

```diff
@@ -11,41 +11,61 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+A DSL built to help poker players keep track of data at the poker table in real time. This dsl would help reinvent poker analyis for poker players. Right now player rely on their memory to write down what happened on any given hand durring game. They may wait till break to add a particular interesting hand into thier notes, or keep a mental note of how a player plays durring a hand, or even wait many hours until the event has stopped or they got busted out to write down things that happened at the table all off of memory. Instead image a DSL for poker players that allows them to keep track of poker related events at the poker table, and even provide them with statistics so they can make their best move.
+
```

Comment:
Ok this is a cool idea - it sounds like it would get you kicked out of a casino though.